sdk/test-utils/recorder/package.json

```diff
@@ -63,17 +63,24 @@
     "fs-extra": "^8.1.0",
     "nise": "^1.4.10",
     "nock": "^11.7.0",
-    "tslib": "^1.10.0"
+    "tslib": "^1.10.0",
+    "md5": "~2.2.1",
+    "mock-fs": "~4.10.4",
+    "mock-require": "~3.0.3"
```

Comment:
these two mock deps should be devdeps right? I don't see them used outside of testing?

Reply:
certainly so! Thank you!